src/js/options.js

```diff
@@ -417,14 +447,14 @@ function refreshFilterPage() {
   $('.tooltip').tooltipster();
 
   // Display tracking domains.
-  var originsToDisplay;
-  var searchText = $("#trackingDomainSearch").val();
-  if (searchText.length > 0) {
-    originsToDisplay = getOriginsArray(searchText);
-  } else {
-    originsToDisplay = allTrackingDomains;
-  }
-  showTrackingDomains(originsToDisplay);
+  // TODO when does this happen?
```

Comment:
[Clarification] Is this an open question still?

Reply:
Just another note to self. Will resolve.